### PR TITLE
Add `--same-password` flag for reading only one password for both files

### PR DESCRIPTION
**Pull request by Tezkerek**

### CODE REVIEW

**Review by Narigo — CHANGES_REQUESTED** [review]
Hi @Tezkerek thanks for this PR!

I had some trouble initially why you want to add a `--same-password` flag as there is `--passwords` already to set both. After finally understanding that one prompt instead of two makes totally sense, this is a really good idea and thanks for taking the time implementing this feature!

If you have time to help out more, you could add the new thing to the Readme, fix the formatting issue, add a test and update the package version to 1.1.0. I can then test it, merge and publish a new version directly.

File: src/main.rs [hunk 1]
```diff
@@ -64,6 +64,12 @@ fn main() -> Result<()> {
         .help("Sets the password for both files (if it's the same for both files)")
         .takes_value(true),
     )
+    .arg(
+      Arg::with_name("same-password")
+      .long("same-password")
+      .help("Asks for password only once, and tries to open both files with it")
+      .takes_value(false),
```
[the remark this answers is not in the document]
Comment (Tezkerek) [reply src/main.rs:71]: Yeah, I forgot to undo that while debugging why my indent detection plugin was setting the indent to 2 spaces.  The rest of the file is indented with 4 spaces, but lines 23-108 are indented with only 2? And rustfmt doesn't seem to care about the indentation in parameters?

**Comment by Tezkerek** [comment]
I don't see any tests for `keepass-diff <file-a> <file-b>`, should the test just check that opening the databases doesn't fail?

**Review by Narigo — APPROVED** [review]
Thanks for adding the documentation and updating the version!

**Comment by Narigo** [comment]
> I don't see any tests for `keepass-diff <file-a> <file-b>`; should the test just check that opening the databases doesn't fail?

Ahh, you're right. I guess I haven't added anything yet for the ones relying on user inputs. I've tested it manually now, it works - so I'll merge it. It looks like it's possible to pipe input into this but then the output does not start on a new line and I'd need to somehow change that. I'll merge this PR now and add an issue for this. Thanks again for the contribution!

**Pull request merged**